suse_migration_services/units/prepare.py

```diff
@@ -162,13 +172,14 @@ def main():
         system_mount.add_entry(
             zypp_plugins_services, '/usr/lib/zypp/plugins/services'
         )
-        if os.path.exists(cloud_register_metadata):
-            log.info('Bind mounting /var/lib/cloudregister')
-            Path.create('/var/lib/cloudregister')
+        if cloud_register_metadata and os.path.exists(cloud_register_metadata):
+            log.info('Bind mounting /var/cache/cloudregister from {0}'.format(
```

Comment:
same here, 
```python
log.info(
       'Bind mounting /var/cache/cloudregister from {0}'.format(
             cloud_register_metadata
       )
)
```
otherwise, let's try to keep it consistent with the previous log
```python
log.info(
       'Bind mounting /var/cache/cloudregister from {0}'.format(
             cloud_register_metadata)
)
```

Reply:
will do